modules/nvidia-gpu-aws-adding-a-gpu-node.adoc

```diff
@@ -8,22 +8,11 @@
 
 You can copy and modify a default compute machine set configuration to create a GPU-enabled machine set and machines for the AWS EC2 cloud provider.
 
-The following table lists the validated instance types:
-
-[cols="1,1,1,1"]
-|===
-|Instance type |NVIDIA GPU accelerator |Maximum number of GPUs |Architecture
-
-|`p4d.24xlarge`
-|A100
-|8
-|x86
-
-|`g4dn.xlarge`
-|T4
-|1
-|x86
-|===
+For more information about the supported instance types, see:
```

Comment:
```suggestion
For more information about the supported instance types, see:
```
- Revise the lead-in to be a complete sentence. Per the ISG, "use a sentence to introduce a list that is not a procedure": https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording
-